text/0013-code-formatters.md

```diff
@@ -0,0 +1,23 @@
+# Code formatters
+- Start Date: Mid 2016
+- RFC PR: https://github.com/CanopyTax/frontend-rfc/pull/13
+
+## Squad vs Chapter vs Frontend-wide
+Each squad may choose whether to use a code formatter and, if so, which one to use.
+
+## Definition
+A code formatter is a tool that changes source code to have consistent indentation, styling, and formatting.
+
+## Reason for decision
+- There has traditionally been a difference in opinion about code formatters, so making it autonomous to squads is a decent compromise.
+  For example, there were some feisty conversations where it was compared to a school uniform by some, and an industry standard by others :).
+- The formatting and styling of code wasn't deemed important enough to enforce at a company wide level.
+
+## Alternatives
+- We could enforce the use of [Prettier](https://github.com/prettier/prettier) or another code formatter.
+
+## Common practices (not enforced)
+- If your squad is using a code formatter, consider adding a [pre-commit hook](https://prettier.io/docs/en/precommit.html) that automatically runs the code formatter.
+  This makes it so people don't have to remember to run it everytime they push code.
+- If your squad is using a code formatter, consider having the build fail if the source code does not match the formatted code.
+  See [Prettier's Integration with eslint](https://prettier.io/docs/en/eslint.html) for more details.
```

Comment:
Full disclosure, I want this change because it's sort of what I already do and would love it if others avoided formatting code I own.

Consider adding something about following the norms of another squad when working in a project they own.
Ex. Squad A uses Prettier and Squad B does not. A member of Squad A should avoid using prettier on Squad B's project and when a member of Squad B does something in a Squad A project the pre-commit hook should run the formatter in an ideal situation.

Reply:
👍 I agree